src/vision/cfg/launch_params_out.yaml

```diff
@@ -0,0 +1,6 @@
+h_high: 318
```

Comment:
We should probably add a comment to this file explaining what these params are for

Reply:
Done